domain-model-assistant/Assets/Components/Scripts/Edge.cs

```diff
@@ -28,6 +28,9 @@ public string ID
     bool hold = false;
     private Diagram _diagram;
     public Vector3 mousePos;
+    public Vector3 linePosition1;
+    public Vector3 linePosition2;
```

Comment:
How do we know which one is which?

Reply:
it's not neccessary to distinguish the two points here since they are just placeholder for positions and don't carry information on the edge ends.